### PR TITLE
Bump 2020.3 version to 2020.3.3

### PR DESCRIPTION
**Pull request by PawelLipski**

### CODE REVIEW

File: frontendResourceBundles/src/main/resources/GitMacheteBundle.properties [hunk 1]
```diff
@@ -223,7 +223,7 @@ string.GitMachete.error-handler.report-action-text=Open Issue On GitHub
 
 string.GitMachete.BranchOrCommitCellRendererComponent.inferred-fork-point.commit=commit
 string.GitMachete.BranchOrCommitCellRendererComponent.inferred-fork-point.fork-point=fork point ?
-string.GitMachete.BranchOrCommitCellRendererComponent.inferred-fork-point.found-in-reflog=has been found in reflog of
+string.GitMachete.BranchOrCommitCellRendererComponent.inferred-fork-point.found-in-reflog=seems to be a part of the unique history of
```
Comment (PawelLipski) [inline frontendResourceBundles/src/main/resources/GitMacheteBundle.properties:226]: I think that's gonna be more user-friendly

File: frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/dialogs/DeleteBranchOnSlideOutSuggestionDialog.kt [hunk 2]
```diff
@@ -31,61 +31,60 @@ class DeleteBranchOnSlideOutSuggestionDialog(project: Project) :
 
   override fun createActions(): Array<Action?> = emptyArray()
 
-  override fun createCenterPanel() =
```
Comment (mkondratek) [inline frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/dialogs/DeleteBranchOnSlideOutSuggestionDialog.kt:34]: what happened here?

File: frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/dialogs/OverrideForkPointDialog.kt [hunk 3]
```diff
@@ -37,46 +37,43 @@ class OverrideForkPointDialog(
         }
       } else null
 
-  override fun createCenterPanel() =
```
Comment (mkondratek) [inline frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/dialogs/OverrideForkPointDialog.kt:40]: :confused:

File: frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/dialogs/SlideInDialog.kt [hunk 4]
```diff
@@ -44,72 +44,66 @@ class SlideInDialog(
   fun showAndGetBranchName() =
       if (showAndGet()) SlideInOptions(branchName.trim(), reattach) else null
 
-  override fun createCenterPanel() =
```
Comment (mkondratek) [inline frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/dialogs/SlideInDialog.kt:47]: ???

File: frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/dialogs/DeleteBranchOnSlideOutSuggestionDialog.kt (hunk 2 above)
Comment (PawelLipski) [reply frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/dialogs/DeleteBranchOnSlideOutSuggestionDialog.kt:34]: It looks that **spotless** likes to change Kotlin style with every minor release... that's not the first time I see random changes like that

File: build.gradle [hunk 5]
```diff
@@ -24,32 +24,32 @@ ext {
...
-    latestStable: '2020.3.2',
+    latestStable: '2020.3.3',
     // Note that we have to use a "fixed snapshot" version X.Y.Z-EAP-SNAPSHOT (e.g. 211.4961.33-EAP-SNAPSHOT)
     // rather than a "rolling snapshot" X-EAP-SNAPSHOT (e.g. 211-EAP-SNAPSHOT)
     // to ensure that the builds are reproducible.
     // EAP-CANDIDATE-SNAPSHOTs can be used for binary compatibility checks,
     // but for some reason aren't resolved in UI tests.
     // Generally, see https://www.jetbrains.com/intellij-repository/snapshots/ -> Ctrl+F idea
-    eapOfLatestSupportedMajor: '211.6693.14-EAP-SNAPSHOT'
+    eapOfLatestSupportedMajor: null //'211.6556.14-EAP-SNAPSHOT'
```
Comment (PawelLipski) [inline build.gradle:34]: **To be restored before this PR is merged**, just trying to get the UI tests to pass...

**Review by mkondratek — APPROVED** [review]
Lgtm

**Pull request merged**